ajax/libs/artDialog/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "artDialog",
+  "filename": "dialog-min.js",
+  "version": "6.0.4",
+  "description": "经典的网页对话框组件，内外皆用心雕琢",
+  "homepage": "https://aui.github.io/artDialog/",
+  "keywords": [
+    "dialog",
+    "ui",
+    "jquery"
+  ],
+  "author": "aui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aui/artDialog.git"
+  },
+  "license": "LGPL",
```

Comment:
This is not precise. You should take a look at [tools/license-list.json](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json).